cmd/debugger/main.go

```diff
@@ -349,7 +376,7 @@ func main() {
 
 			err := os.Setenv("TERM", debuggerSettings.Term)
 			if err != nil {
-				conslogger.Warnf("Failed to set term: %v", err)
+				conslogger.VerboseWarnf("Failed to set term: %v\n", err)
```

Comment:
Was this warning commonly occurring? I would think having `Setenv` fail should be extremely rare, and should always show up as a warning -- if the TERM can't be set, feature like resizing the terminal won't work.

Reply:
sounds good.